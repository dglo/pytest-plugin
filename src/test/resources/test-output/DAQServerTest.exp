tsts=5
errs=0
fail=0
time=6.001
xml=false
