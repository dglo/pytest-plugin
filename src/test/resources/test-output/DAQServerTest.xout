option -x not recognized
Usage: DAQServerTest.py [options] [test] [...]

Options:
  -h, --help       Show this message
  -v, --verbose    Verbose output
  -q, --quiet      Minimal output

Examples:
  DAQServerTest.py                               - run default set of tests
  DAQServerTest.py MyTestSuite                   - run suite 'MyTestSuite'
  DAQServerTest.py MyTestCase.testSomething      - run MyTestCase.testSomething
  DAQServerTest.py MyTestCase                    - run all 'test*' test methods
                                               in MyTestCase

