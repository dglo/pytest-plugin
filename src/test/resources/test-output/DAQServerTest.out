testLogFallback (__main__.TestDAQServer) ... Start of log at localhost:12345
End of log
log - - [2008-09-08 11:48:26.869023] Start of log at localhost:12345
log - - [2008-09-08 11:48:26.869023] Start of log at localhost:12345
log - - [2008-09-08 11:48:26.869124] End of log
log - - [2008-09-08 11:48:26.869124] End of log
Start of log at localhost:456778
End of log
new - - [2008-09-08 11:48:26.883501] Start of log at localhost:456778
new - - [2008-09-08 11:48:26.883501] Start of log at localhost:456778
new - - [2008-09-08 11:48:26.883565] End of log
new - - [2008-09-08 11:48:26.883565] End of log
Start of log at localhost:12345
log - - [2008-09-08 11:48:26.889685] Start of log at localhost:12345
log - - [2008-09-08 11:48:26.889685] Start of log at localhost:12345
End of log
log - - [2008-09-08 11:48:27.391916] End of log
log - - [2008-09-08 11:48:27.391916] End of log
ok
testNoRunset (__main__.TestDAQServer) ... ok
testRegister (__main__.TestDAQServer) ... ok
testRegisterWithLog (__main__.TestDAQServer) ... ok
testRunset (__main__.TestDAQServer) ... ok

----------------------------------------------------------------------
Ran 5 tests in 6.001s

OK
Got registration for ID#1 foo#0 at localhost:666
Start of log at localhost:123
Got registration for ID#2 foo#0 at localhost:666
End of log
Got registration for ID#3 foo#0 at localhost:666
Built runset with the following components:
ID#3 foo#0 at localhost:666

ID#3 foo#0 at localhost:666 connected
Start of log at abc:777
Version info: unknown unknown unknown unknown unknown trunk 3472:3474M
End of log
Start of log at abc:777
